src/XComponentAPI.js

```diff
@@ -0,0 +1,70 @@
+
+define(["Connection", "Parser", "GetterXcApi"], function (Connection, Parser, GetterXcApi) {
+    "use strict";
+
+    var Factory = function () {
```

Comment:
Remove the factory. Use a simple factory method create
=> easier to use the api (only 1 step)
xcomponentApi.create(serverUrl)